tools/version.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <PropertyGroup>
+    <Authors>Henrik Feldt, Logibit AB, formerly @mausch</Authors>
```

Comment:
If you are moving to msbuild 15 all projects..

if you move this file in `src` directory, and rename it `Directory.Build.props`, is automatically imported all msbuild project in each subdirectory.
So you can remove `<Import Project="..\..\tools\version.props" />`

That's [a new msbuild 15 feature](https://docs.microsoft.com/en-us/visualstudio/msbuild/what-s-new-in-msbuild-15-0).
Pratically each `fsproj` search recursively in the directory hierarchi for a `Directory.Build.props` (and `Directory.Build.targets` too) to import.